main/main.cpp

```diff
@@ -124,17 +149,37 @@ void app_main(void)
 	};
 	esp_timer_handle_t timer_handle;
 	esp_timer_create(&timer_config, &timer_handle);
-	esp_timer_start_periodic(timer_handle, SAMPLE_PERIOD_us);
+	esp_timer_start_periodic(timer_handle, SAMPLE_TIME_us);
 
 	ControllerTask controllerTask(
-		"Controller Task", 2400,
+		"Controller Task", 2800,
```

Comment:
weird number  for a task size. they are almost 2^N  values. For example candidate could be 4096